refresh/index.js

```diff
@@ -1074,6 +1081,7 @@ module.exports = Generator.extend({
             this.templatePath('fromswagger', 'Routes.swift.hbs'),
             this.destinationPath('Sources', this.applicationModule, 'Routes', `${resource}Routes.swift`),
             {
+              services: Object.keys(this.appInitCode.services).length > 0,
```

Comment:
I couldn't see what this is used for...

Reply:
the services property always contains a dictionary object. If it is empty, then I wont generate the ApplicationServices class and the call to init it

Reply:
I'm confused. `Routes.swift.hbs` doesn't seem to use a variable called `services`.

Reply:
hmmm, what can I say :-o